lib/web/ui/server.go

```diff
@@ -0,0 +1,61 @@
+package ui
+
+import (
+	"sort"
+
+	"github.com/gravitational/teleport/lib/services"
+)
+
+// LabelDTO describes a label
+type LabelDTO struct {
+	// Name is this label name
+	Name string `json:"name"`
+	// Value is this label value
+	Value string `json:"value"`
+}
+
+// ServerDTO describes a server
+type ServerDTO struct {
+	// Name is this server name
+	Name string `json:"id"`
+	// ClusterName is this server cluster name
+	ClusterName string `json:"siteId"`
```

Comment:
should be `clusterName`, I think you should rename this in the UI as we are deprecating `site` everywhere

Reply:
I agree but I would like to do it separately from this change